packages/opentelemetry-resources/src/types.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Resource } from './Resource';
+
+export interface Labels {
```

Comment:
Do you think we should rename this to `ResourceLabels` (or something like this), to avoid confusion between metrics [Labels](https://github.com/open-telemetry/opentelemetry-js/blob/5d6c99d126a8197494968ce8430a05decc0afee9/packages/opentelemetry-api/src/metrics/Metric.ts#L138)?

Reply:
Yes this should be renamed